### DISCUSSION TITLE
A small addition to the .gitignore file

### DISCUSSION THREAD

**Discussion by krasimir** [reactjs/reactjs.org-translation#242] [Team Posts]
Hey all,

do you mind if I add `.vscode` to the `.gitignore` file. I want to exclude some stuff while translating.

**Comment by tesseralis** [comment]
I think that should be fine. Do you want to make a pull request to the main reactjs.org repo?

**Comment by krasimir** [comment]
Yep, that will be awesome. I bet lots of people are using Code and that's a handy option to set settings specifically per project.

**Comment by krasimir** [comment]
Just to give you an example: I have a prettier extension that formats the code but I wasn't able to turn it off for bg.reactjs.org folder. As far as I know the only one way to do that and keep that folder in the same workspace as my other projects is to add a custom `.vscode/settings.json` file. For the time being I'll add this folder but will not commit it.

**Comment by eduardomoroni** [comment]
I'm Ok with that. But if most people don't like it you can ignore some files locally.
https://stackoverflow.com/questions/1753070/how-do-i-configure-git-to-ignore-some-files-locally

**Comment by krasimir** [comment]
@eduardomoroni oh nice. I didn't know that approach. I actually solved it by creating different workspaces in my editor. But that's a good option too. Thanks.
